server/graphql/v2/collection/TagStatsCollection.js

```diff
@@ -0,0 +1,18 @@
+import { GraphQLList, GraphQLObjectType } from 'graphql';
+
+import { Collection, CollectionFields } from '../interface/Collection';
+import { TagStats } from '../object/TagStats';
+
+export const TagStatsCollection = new GraphQLObjectType({
+  name: 'TagCollection',
```

Comment:
Why the mismatch between `TagStatsCollection` and `TagCollection`?

Reply:
Sorry, mistake on my part when typing. I've corrected this. 🤔